metabrainz/templates/users/profile.html

```diff
@@ -18,10 +18,50 @@
     </span>
   </p>
 
+  {% if current_user.is_commercial %}
+      {% if current_user.state == 'active' and current_user.good_standing %}
```

Comment:
That will never really be seen by outsiders. Someone is in the deadbeat club because we added them to it. When they decide to start supporting us, I'll move them to the correct tier before I change the user.